### ISSUE TITLE
Adding summary variable can mess up combo box display in variable grid

### ISSUE THREAD

**Issue by vchelaru** [vchelaru/FlatRedBall#1618]
1. Create or open a project that has a Top Down player (use the wizard if you don't have one)
2. Expand the Player variables
3. Select one of the movement variables
![image](https://github.com/user-attachments/assets/72c96797-61c4-48ac-8e0f-a36320430dde)
4. Enter a summary - it should have spaces and a few words
![image](https://github.com/user-attachments/assets/ae568745-403e-4fc5-b5ea-bd6af03016c6)
5. Select the Variable folder on the entity
![image](https://github.com/user-attachments/assets/bc338096-b601-401d-8d17-e27aceb4658f)

Observe - the variable that had text added to it is shrunk
Expected - it should stay the same size

This control is defined in ComboBoxDisplay.cs in the Gum project. There's comments there explaining why it's done in C# code rather than XAML but if it can work in XAML, that would be fine too.

**Comment by vchelaru** [comment]
More info on this issue:

@vicdotexe looked into it and said:

> telling that summary to only span 1 column seems to fix it and doesn't matter what column it's in, horizontalalignment on the third textbox doesn't appear to have an affect.

> I'm gonna chalk it up to threading the needle in some way that causes the internal grid calculations to go haywire and that it's not actually implemented incorrectly... definitely something related to the column span, but nothing popping out at me that actually makes sense why the column span.

As I've looked into this there are actually 2 problems:
1. Changing the summary modifies the size of the combo box (as shown in the pic above)
2. The summary can get long enough as to force a scroll bar, and really long summaries can make the variable grid unusable.

I have hacked a workaround:

1. By putting the summary in a stack view, the combo box no longer gets affected by the size of the summary. This fixes the original issue posted above
2. By putting a max width on the summary of 250 pixels, that gives a good balance between being long enough to not wrap too often, but not so short as to stack a ton.

This is good enough for now, since it was somewhat blocking Cthulhu.
